polyfill/test/usercalendar.mjs

```diff
@@ -293,6 +293,9 @@ describe('Userland calendar', () => {
       },
       fields(fields) {
         return fields.slice();
+      },
+      mergeFields(fields, extraFields) {
+        return Object.assign(fields, extraFields);
```

Comment:
Isn't this where you want to have the default logic that handles monthCode, era, and eraYear?

Reply:
Correct, but none of those are implemented in the polyfill yet, so there's nothing else for this method to do at this time. That will be added later.